packages/core/src/amazonq/lsp/lspController.ts

```diff
@@ -58,9 +60,9 @@ export interface Manifest {
         targets: Target[]
     }[]
 }
-const manifestUrl = 'https://aws-toolkit-language-servers.amazonaws.com/q-context/manifest.json'
+const manifestUrl = 'https://ducvaeoffl85c.cloudfront.net/manifest-0.1.40.json'
```

Comment:
I assume this intentional and we will revert it before releasing?

Reply:
Absolutely. We will use the public endpoint, this is to be fixed by next Monday morning